src/Symfony/Bundle/FrameworkBundle/Resources/config/debug_prod.xml

```diff
@@ -14,7 +14,7 @@
         <service id="debug.debug_handlers_listener" class="%debug.debug_handlers_listener.class%">
             <tag name="kernel.event_subscriber" />
             <tag name="monolog.logger" channel="php" />
-            <argument /><!-- Exception handler -->
+            <argument>null</argument><!-- Exception handler -->
```

Comment:
Yes, it is handled this way in [XmlUtils:phpize()](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Config/Util/XmlUtils.php#L184-L217).